src/main/java/io/cdap/plugin/http/source/common/exceptions/InvalidPropertyTypeException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
```

Comment:
Use proper year

Reply:
I copied this from the google drive plugin. Do I just update the year?

Reply:
yep